README.markdown

```diff
@@ -5,29 +5,57 @@ Puppet Dashboard is a web interface for [Puppet](http://www.puppetlabs.com/).
 It can view and analyze Puppet reports, assign Puppet classes and parameters to
 nodes, and view inventory data and backed-up file contents.
 
-For full documentation, see the [Puppet Dashboard Manual](http://docs.puppetlabs.com/dashboard/manual).
-
 Dependencies
 ------------
 
-* Ruby 1.8.7, 1.9.3 or 2.0.0
+* Ruby 1.8.7, 1.9.3, 2.0.0 or 2.1.x
```

Comment:
was there a yaml lib change in ruby 2.1 that necessitates some other changes in the project?

Reply:
Don't know. Just noticed that Debian Jessie has Ruby 2.1 installed and that the dashboard worked with it.

Reply:
I think it must be 2.2; I was trying to configure it to work under 2.2 and ran into that.